tests/collections/collection.test.ts

```diff
@@ -397,6 +476,20 @@ describe(`StargateMongoose - ${testClientName} Connection - collections.collecti
             assert.strictEqual(findResDocs.length, 1);
             assert.strictEqual(findResDocs[0]._id, idToCheck);
         });
+        it('should find & findOne any level String NE $ne document', async () => {
+            const doc1 = createSampleDocWithMultiLevel();
+            const doc2 = createSampleDoc2WithMultiLevel();
+            const insertDocResp1 = await collection.insertOne(doc1);
+            const insertDocResp2 = await collection.insertOne(doc2);
+            const idToCheck2 = insertDocResp2.insertedId;
+            const filter = { 'address.street': { '$ne': doc1.address?.street } };
```

Comment:
I'd also recommend testing `{ 'address.street': { '$ne': doc2.address?.street } }` just to reduce risk of tests passing by accident

Reply:
Thanks, solved!